packages/odyssey-react-mui/src/labs/GroupPicker.tsx

```diff
@@ -218,9 +204,8 @@ const GroupPicker = <
       // AutoComplete is wrapped in a div within MUI which does not get the disabled attr. So this aria-disabled gets set in the div
       aria-disabled={isDisabled}
       data-se={testId}
-      disableCloseOnSelect={hasMultipleChoices}
```

Comment:
Close the options list after selected even for multiple choices, that matches the existing behavior.